hibernate-core/src/main/java/org/hibernate/type/spi/TypeConfiguration.java

```diff
@@ -108,6 +109,7 @@
  * @see org.hibernate.boot.model.TypeContributions
  */
 @Incubating
+@SuppressWarnings( "nullness" )
```

Comment:
I guess you added that because you figured out it is hard to properly annotate this class for nullness? If you have to, I'd prefer if you do that on a per-method basis instead of for the whole class.